### PR TITLE
feat(tags): support removable

### PR DESCRIPTION
**Pull request by yinonov**
issues to resolve:

- [ ] tags wrapper to observe assigned nodes (measure performance of textContent tags instances / manage state of assigned tags) @yonatankra
- [ ] consider removing dense / enlarge if not required at this point @jshenkman
- [ ] research focus management in accordance to keyboard control @yinonov
- [ ] should remove button feature ripple as well? @rachelbt
- [ ] should selected feature ripple? @rachelbt

image

### CODE REVIEW

**Comment by yinonov** [comment]
will handle tag open issues in a different scope

File: components/tags/src/vwc-tag.scss [hunk 1]
```diff
@@ -126,3 +128,11 @@
 // 	}
 // }
 
+.remove-button {
+	all: unset;
+	margin-inline-start: 10px;
+
+	.vwc-tag__icon {
+		--vvd-icon-size: 10px;
```
Comment (rachelbt) [inline components/tags/src/vwc-tag.scss:136]:
```suggestion
		--vvd-icon-size: 12px;
```

File: components/tags/src/vwc-tag.scss [hunk 2]
```diff
@@ -83,6 +83,7 @@
 
 	&.vwc-tag--selected {
 		&.layout-outlined {
+			border-color: var(#{layout.$vvd-layout-color-outline});
```
Comment (rachelbt) [inline components/tags/src/vwc-tag.scss:86]: according to the figma it is not the right color:
image
and in the code:
image

**Review by rachelbt — COMMENTED** [review]
there's no option for both selected & removable. Is it intentionally?
There's no removable option in the UI test

File: components/tags/src/vwc-tag.scss (hunk 2 above)
Comment (yinonov) [reply components/tags/src/vwc-tag.scss:86]: given that this is in discussions with design and partially handled in vonage/vivid-3#389
can this be resolved for now?

**Comment by yinonov** [comment]
> there's no option for both selected & removable. Is it intentionally? There's no removable option in the UI test

actually that's a good question. I don't think they should co-exist but it's not specified. let's try to monitor public feedback and "field" testing. semantics & a11y concerns should be researched here

**Comment by rachelbt** [comment]
I think we need to add wither just cursor pointer to the button or both that & ripple to indicate that the X is a button and not decorative

**Comment by yinonov** [comment]
> I think we need to add wither just cursor pointer to the button or both that & ripple to indicate that the X is a button and not decorative

It is addressed in different issues

**Pull request merged**